### DISCUSSION TITLE
"unwrap-intf.csh"command no found,and ""unwrap_parallel.csh“parallel: Command not found.

### DISCUSSION THREAD

**Discussion by endlinene** [gmtsar/gmtsar#744] [Q&A]
`gmtsar-user@gmtsarvb:~/Downloads/des/F1/intf_all$ unwrap_intf.csh intflist 

unwrap_intf.csh: command not found

gmtsar-user@gmtsarvb:~/Downloads/des/F1/intf_all$ unwrap_parallel.csh intflist  1

parallel: Command not found.
Finished all unwrapping jobs...

`

**Comment by Xiaohua-Eric-Xu** [comment]
unwrap_intf.csh is a script that needs to be customized by users.

Parallel not found could be because it was not installed with GMTSAR. You could try `sudo apt install parallel`.
